packages/cli/test/commands/migrate/e2e.ember-addon.test.ts

```diff
@@ -0,0 +1,276 @@
+      // expect(
+      //   readFileSync(resolve(addonComponentsPath, 'branch.gts'), { encoding: 'utf-8' })
+      // ).toMatchSnapshot();
+      // expect(
+      //   readFileSync(resolve(addonComponentsPath, 'leaf.gts'), { encoding: 'utf-8' })
+      // ).toMatchSnapshot();
+      // expect(
+      //   readFileSync(resolve(addonComponentsPath, 'trunk.gts'), { encoding: 'utf-8' })
+      // ).toMatchSnapshot();
+    });
+
+    test('should have the correct files in the config directory', () => {
+      const configPath = resolve(project.baseDir, 'config');
+      const configFiles = readdirSync(configPath);
+      expect(configFiles).toContain('ember-try.js');
+      expect(configFiles).toContain('environment.js');
+    });
+
+    test('should have the correct files in the tests directory', () => {
+      const testsPath = resolve(project.baseDir, 'tests');
+      const testsFiles = readdirSync(testsPath);
+      expect(testsFiles).toContain('test-helper.ts');
+      expect(testsFiles).toContain('index.html');
+      // let testHelperContent = readFileSync(resolve(testsPath, 'test-helper.ts'), {
+      //   encoding: 'utf-8',
+      // });
+      // testHelperContent = testHelperContent.replace(
+      //   new RegExp(`${project.baseDir}`, 'g'),
+      //   '<tmp-path>'
+      // );
+      // expect(testHelperContent).toMatchSnapshot();
+    });
+
+    test('should have the correct files in the tests dummy app directory', () => {
+      const testsDummyAppPath = resolve(project.baseDir, 'tests', 'dummy', 'app');
+      const testsDummyAppFiles = readdirSync(testsDummyAppPath);
+      expect(testsDummyAppFiles).toContain('app.ts');
+      expect(testsDummyAppFiles).toContain('router.ts');
+      expect(testsDummyAppFiles).toContain('index.html');
+      // let appContent = readFileSync(resolve(testsDummyAppPath, 'app.ts'), { encoding: 'utf-8' });
+      // appContent = appContent.replace(new RegExp(`${project.baseDir}`, 'g'), '<tmp-path>');
+      // let routerContent = readFileSync(resolve(testsDummyAppPath, 'router.ts'), {
+      //   encoding: 'utf-8',
+      // });
+      // routerContent = routerContent.replace(new RegExp(`${project.baseDir}`, 'g'), '<tmp-path>');
+      // expect(appContent).toMatchSnapshot();
+      // expect(routerContent).toMatchSnapshot();
+    });
+
+    test('should have the correct files in the tests dummy app templates directory', () => {
+      const testsDummyTemplatesPath = resolve(
+        project.baseDir,
+        'tests',
+        'dummy',
+        'app',
+        'templates'
+      );
+      const testsDummyTemplatesFiles = readdirSync(testsDummyTemplatesPath);
+      expect(testsDummyTemplatesFiles).toContain('application.hbs');
+      // expect(
+      //   readFileSync(resolve(testsDummyTemplatesPath, 'application.hbs'), { encoding: 'utf-8' })
+      // ).toMatchSnapshot();
+    });
+  });
+});
+
+function verifyBaseDir(basePath: string): void {
```

Comment:
since this PR is focused on `e2e migrate tests for ember apps` this function is largely focused on the contents of *config* files which is covered in the config tests. we can assume that is working if passing in the other tests and we dont need the additional overhead here. 

primarily in this function we care about:

```
  expect(fileList).toContain('ember-cli-build.js');
  expect(fileList).toContain('testem.js');
  expect(fileList).toContain('.ember-cli');
  expect(fileList).toContain('index.js');
```

which the function declaration should be renamed to match what we are doing eg.
```
function expectFilesExist(basePath, [files]) {
   const fileList = readdirSync(basePath);

   // iterate over the files
   // for ...
   expect(fileList).toContain(file);
}
```

and the function expression would clearly show the files being expected (which is easier to read).
```
const emberAddonFiles = ['ember-cli-build.js', 'testem.js'...];
expectFilesExist('foo', emberAddonFiles);
```

Reply:
done